Some improvements to plugins and projects documentation (#3702)
geosolutions-it/MapStore2: docs/developer-guide/mapstore-projects.md

# MapStore Projects
MapStore projects can be created using the [Project Creation Script](project-creation-script).

> A MapStore project is an application using the MapStore framework to build a custom WebGis application.

Reviewer: ```suggestion
A MapStore project is a custom WebGis application that uses MapStore as a framework.
```
